chrisgreg/bloom
# test/javascript interop

PR raised purely for testing.

Noone wants a component that just `alert`s on mount!  

## Testing 

- Run Storybook in `./bloom-site` 
- Go to http://localhost:4000/storybook/core_components/alert
- Observe that a Javascript alert is triggered
